lib/cadet_web/router.ex

```diff
@@ -79,6 +79,7 @@ defmodule CadetWeb.Router do
     post("/assessments/question/:questionid/answer", AnswerController, :submit)
 
     get("/achievements", IncentivesController, :index_achievements)
+    get("/achievements/totalxp", IncentivesController, :calculate_totalxp)
```

Comment:
This probably makes more sense as `/user/total_xp` because the XP value includes assessment XP as well.